volatility3/framework/plugins/regexscan.py

```diff
@@ -46,38 +46,50 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
             ),
         ]
 
-    def _generator(self, regex_pattern):
-        regex_pattern = bytes(regex_pattern, "UTF-8")
-        vollog.debug(f"RegEx Pattern: {regex_pattern}")
+    def _generator(self, layer, pattern, maxsize):
```

Comment:
```suggestion
    def _generator(self, context, layer_name, pattern, maxsize):
        layer = self.context.layers[layer_name]
```
Sorry to be a pain, but we prefer people to pass the context and then basic variables (like strings) to look up things like layers.  This makes the whole thing easier to parcel up for if we ever want to move to a multithreaded system.